bootloader/efi/efi.go

```diff
@@ -89,11 +89,10 @@ func openEFIVarImpl(name string) (r io.ReadCloser, attr VariableAttr, size int64
 		return nil, 0, 0, fmt.Errorf("unexpected size: %d", sz)
 	}
 
-	var abuf [4]byte
-	if _, err = varf.Read(abuf[:]); err != nil {
+	if err = binary.Read(varf, binary.LittleEndian, &attr); err != nil {
```

Comment:
Nitpick: it it's immediately obvious what is the size of `attr` (or `VariableAttr`), this might make it a bit more clear:
```
var attr32 uint32
if err = binary.Read(varf, binary.LittleEndian, &attr32); err != nil {
...
}
return varf, VariableAttr(attr32), sz - 4, nil
```